lib/raygun.offline.ts

```diff
@@ -30,18 +27,30 @@ export class OfflineStorage {
       err,
       cacheContents
     ) {
-      raygunTransport.send(JSON.parse(cacheContents));
+      if (!storage.transport) {
+        throw new Error("OfflineStorage was initialized without transport");
+      }
```

Comment:
Would it be better for this check to be included in the OfflineStorages `init` method?

Reply:
I suppose it shouldn't matter too much because this is more of an internal class

Reply:
Good point! This error check only existed to please the compiler, since the transport might be undefined if we forgot to call init.

I've refactored this so that the transport is passed in via the constructor, so we don't need to worry about it being undefined inside of that class anymore.